files/docker-image-prune.service

```diff
@@ -0,0 +1,11 @@
+# SPDX-FileCopyrightText: 2025 Stefan Haun <tux@netz39.de>
+# SPDX-License-Identifier: MIT
+
+[Unit]
+Description=Prune unused Docker images
+Requires=network-online.target docker.service
+After=network-online.target docker.service
+
+[Service]
+Type=oneshot
+ExecStart=/usr/bin/docker iamge prune --all --force
```

Comment:
Typo!

Reply:
Man, I fixed that - but apparently didn't commit and managed to throw the fix away. (Because it is fixed on the target system and worked.)

Reply:
This is an argument for test cases in the role.